packages/mui-material/src/Slider/Slider.js

```diff
@@ -523,55 +592,195 @@ const Slider = React.forwardRef(function Slider(inputProps, ref) {
   const markLabelSlotProps = slotProps?.markLabel ?? componentsProps.markLabel;
   const inputSlotProps = slotProps?.input ?? componentsProps.input;
 
+  const rootProps = useSlotProps({
+    elementType: RootSlot,
+    getSlotProps: getRootProps,
+    externalSlotProps: rootSlotProps,
+    externalForwardedProps: other,
+    additionalProps: {
+      ...(shouldSpreadAdditionalProps(RootSlot) && {
+        as: component,
+      }),
+    },
+    ownerState: {
+      ...ownerState,
+      ...rootSlotProps?.ownerState,
+      color,
+      size,
```

Comment:
```suggestion
```

Aren't these already included in the `ownerState`?

Reply:
Yup they are. Nice catch!